tests/testthat/test-prepend.R

```diff
@@ -13,3 +13,27 @@ test_that("prepend is clearer version of merging with c()", {
     }
   )
 })
+
+test_that("prepend appends at the beginning for empty list by default", {
+  x <- list()
```

Comment:
Nitpick: I would write this as `expect_identical(prepend(list(), 1), 1)` for clarity.